.env

```
@@ -0,0 +1,2 @@
+EHR2MEDS_CONFIGS=${PWD}/configs
+EHR2MEDS_DATA=${PWD}/data
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)